src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -194,32 +263,42 @@ export class DatePicker {
   }
 
   render() {
+    const date = <chi-date
+      min={this.min}
+      max={this.max}
+      locale={this.locale}
+      value={this.value}
+      format={this.format}
+      excluded-weekdays={this.excludedWeekdays}
+      excluded-dates={this.excludedDates}
+    />;
+    const time = this.time ? <chi-time /> : null;
+    const popoverContent = this.time ?
+      <div class="-d--flex">
+        {date}
+        {time}
+      </div> : date;
     const chiPopover = (
       <chi-popover
         id="example-4-be-popover"
-        position="bottom"
+        position={this.time ? 'bottom-start' : 'bottom'}
         reference={`#${this._uuid}-control`}
         prevent-auto-hide
         active={this.active}
       >
-        <chi-date
-          min={this.min}
-          max={this.max}
-          locale={this.locale}
-          value={this.value}
-          format={this.format}
-          excluded-weekdays={this.excludedWeekdays}
-          excluded-dates={this.excludedDates}
-        />
+        {popoverContent}
       </chi-popover>
     );
 
     return [
       // TODO: This input should be chi-input in the future and will pass through
       // some of its configuration attributes.
-      <div class={this.disabled ? '-disabled' : ''}>
+      <div class={`
+        ${this.disabled ? '-disabled' : ''}
+        ${this.time ? '-time' : ''}`}>
```

Comment:
If time attribute is specified, can we change placeholder text to `MM/DD/YYYY, --:-- --`